USAGE.md

```diff
@@ -0,0 +1,33 @@
+# How to use Testing Farm as GitHub Action in your projects
+
+Before start of using Testing Farm as GitHub Action in your projects,
+there are several steps which you have to do it.
+
+## As Testing Farm team for public or private ranch
+
+Before any implementation as Testing Farm team for API_KEY here [onboarding](https://docs.testing-farm.io/general/0.1/onboarding.html).
+
+## Setup TMT plan in GitHub / GitLab repos
+
+In your GitHub or GitLab repos create:
+* [.fmf](https://github.com/sclorg/sclorg-testing-farm/tree/main/.fmf) directory with file [version](https://github.com/sclorg/sclorg-testing-farm/blob/main/.fmf/version) which contains 1.
+* [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) directory which will contain TMT plans for Testing Farm
+* create a TMT plan like it is already done here [c9s](https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf)
+* The TMT plan contains instructions how to setup, install and execute tests on quest provided by Testing Farm. For more information see [documentation](https://tmt.readthedocs.io/en/stable/examples.html#plans) or take an inspiration from our [plans](https://github.com/sclorg/sclorg-testing-farm/tree/main/plans) diretory
```

Comment:
Would it be possible to provide an example of how to run the tmt locally for testing the plan? It is in the `tmt` documentation, but a quick-start one-liner might be helpful..

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/64/commits/d3ab80538320df2b1fb56b74361edad681d00dd7